### ISSUE TITLE
Android 的默认文件保存路径

### ISSUE THREAD

**Issue by xtremforce** [LANDrop/LANDrop#13]
这软件收取文件似乎很特殊 (是否临时文件？)，因为接受之后找不到文件的路径的，必须「Share」之后选择文件管理器再去保存。

是否可以像 Mac 一样，设置文件的默认接收路径，直接保存到指定的目录？

**Comment by yvbbrjdr** [comment]
Yup, after I figure out how to manage the permission of writing external storage, I'll be saving the files to the public Download directory.
Android seems to impose a stricter policy since Android 10.

**Comment by yvbbrjdr** [comment]
Also, pls help me reply to this comment in your website:
> windows 下 传文件给 linux 提示
> The proxy type is invalid for this operation 然后闪退了
> 扎心

This will be fixed in the next version.
